src/bridge/graphs/peg_out.rs

```diff
@@ -803,9 +800,16 @@ impl PegOutGraph {
         }
     }
 
-    pub fn connector_1_id(&self) -> ConnectorId { self.connector_1.id() }
-    pub fn connector_2_id(&self) -> ConnectorId { self.connector_2.id() }
-    pub fn connector_6_id(&self) -> ConnectorId { self.connector_6.id() }
+    pub fn connector_0_id(&self) -> ConnectorId { self.connectors.connector_0.id() }
```

Comment:
FYI, we do away with connector IDs in PR #78. They're no longer needed after the Winternitz secret rework.